samples/nrf9160/aws_fota/README.rst

```diff
@@ -29,6 +31,57 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+Prerequisites
+*************
+
+The connected device must have the same MQTT Client ID as the thing name created on AWS IoT.
+It also needs a policy attached to the certificates, and the certificates need to be active.
+
+Below is a very permissive policy which should be used for **testing only** which you can attach to the certificates if you are unsure on what policies are needed for your device. To create a policy go to `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Secure>Policies>Create.
+
+.. code-block:: javascript
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "iot:*",
+            "Resource": "*"
+          }
+       ]
+     }
+
+To create a thing log in to the `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Manage>Things>Create>Create a single thing.
+Give it a name which corresponds to your MQTT Client ID.
+Chose next either add your certificates or chose to create certificates, press activates and remember to download one of the `root CA certificates
```

Comment:
chose -> choose

How about:
"Next. add your certificates, or create new ones by clicking ???."
(I don't know if it's a button you click) :)

Reply:
typos are fixed

For the other part, the sentence is now:
"Next, choose to either create certificates or add your certificates and click **Activate**."

@sigvartmh  , hopefully this is correct.

Reply:
Yeah thanks for spotting :)